src/views/Dashboard/Customer/Home/index.vue

```diff
@@ -28,21 +30,203 @@
+                slot="icon" size="20"
+                color="#C400A5"
+                stroke
+                @click="goToDetail(item)"
+                )
 </template>
 
 <script>
-import { creditCardIcon, layersIcon, labIllustration } from "@/common/icons"
+import { creditCardIcon, layersIcon, labIllustration, doctorDashboardIllustrator, eyeIcon } from "@/common/icons"
+
+import Banner from "@/common/components/Banner"
+import DataTable from "@/common/components/DataTable"
+import dataTesting from "./MyTest/dataTesting.json"
+import Button from "@/common/components/Button"
 
 export default {
   name: "CustomerHome",
 
-  data: () => ({ creditCardIcon, layersIcon, labIllustration, cardBlock: false }),
+  components: {Banner, DataTable, Button },
+  data: () => ({
+    creditCardIcon,
+    layersIcon,
+    labIllustration,
+    eyeIcon,
+    cardBlock: false,
+    orderHistory: [],
+    testHistory: [],
+    titleWording: "",
+    doctorDashboardIllustrator,
+
+    headers: [
+      { text: "Service Name", value: "service_info.name",sortable: true },
+      { text: "Lab Name", value: "lab_info.name", sortable: true },
+      { text: "Date", value: "created_at", sortable: true },
+      { text: "Status", value: "status", sortable: true },
+      {
+        text: "Actions",
+        value: "actions",
+        sortable: false,
+        align: "center",
+        width: "5%"
+      }
+    ]
+  }),
 
   mounted() {
     window.addEventListener("resize", () => {
       if (window.innerWidth <= 959) this.cardBlock = true
       else this.cardBlock = false
     })
+  },
+
+  async created() {
+    await this.getDataOrder()
+    await this.checkOrderLenght()
+  },
+
+  methods: {
+    async getDataOrder() {
+      this.orderHistory = dataTesting.data.map(result => ({
+        ...result._source,
+        id: result._id,
+        updated_at: new Date(parseInt(result._source.updated_at)).toLocaleDateString(),
+        created_at: new Date(parseInt(result._source.created_at)).toLocaleDateString(),
+        timestamp: parseInt(result._source.created_at)
+      }))
+    },
+
+    async getDataTestHistory() {
+      // beda nya apa sama data history, kalo beda dari status ya nanti di filter
+      this.testHistory = dataTesting.data.map(result => ({
+        ...result._source,
+        id: result._id,
+        updated_at: new Date(parseInt(result._source.updated_at)).toLocaleDateString(),
+        created_at: new Date(parseInt(result._source.created_at)).toLocaleDateString(),
+        timestamp: parseInt(result._source.created_at)
+      }))
+    },
+
+    goToOrderHistory() {
+      console.log("to order history")
+      // this.$router.push({ name: "customer-test" }) //go to order history page 
+    },
+    goToDetail() { //item
+      console.log("go to detail")
+      // this.$router.push({ name: "order-history-detail", params: item}) //go to order history detail page
+    },
+
+    async checkOrderLenght() {
+      if (this.orderHistory.length == 0) {
```

Comment:
`!this.orderHistory.length`

Reply:
done. chaenge to !this.orderHistory.length